server/entities/database/juries.js

```diff
@@ -0,0 +1,31 @@
+const sequelize = require('sequelize');
+const connection = require('./connection');
+const Model = sequelize.Model;
+const dataTypes = sequelize.DataTypes;
+
+class Juries extends Model{}
+
+Juries.init({
+    ProjectId: {
```

Comment:
Please ensure all columns follow the conventions defined by the existing ones. Concretely, column names should follow snake case.